src/components/overview/Overview.tsx

```diff
@@ -24,56 +24,43 @@ export interface OverviewRow {
   resourceType?: 'PDF' | 'HTML' | 'JSON' | 'GeoJSON' | 'CSV';
 }
 
-export default function Overview({
-  content,
-  header,
-}: {
-  content: OverviewRow[];
-  header: {
-    name: string;
-    description: string;
-  };
-}) {
+export default function Overview({ content }: { content: OverviewRow[] }) {
   const tableT = useTranslations('Table');
   const columns = [
     {
       cell: (row: OverviewRow) => transformContentToHTMLElement(row),
     },
   ] as TableColumn<unknown>[];
   return (
-    <div className="flex-fill ms-2">
-      <h3>{header.name}</h3>
-      <p className="lead">{header.description}</p>
-      <DataTable
-        dense={true}
-        columns={columns}
-        data={content}
-        noDataComponent={tableT('noRecords')}
-        highlightOnHover
-        pagination={content.length > 10}
-        paginationComponentOptions={{
-          rowsPerPageText: tableT('rowsPerPageText'),
-          rangeSeparatorText: tableT('rangeSeparatorText'),
-          selectAllRowsItem: true,
-          selectAllRowsItemText: tableT('selectAllRowsItemText'),
-        }}
-      />
-    </div>
+    <DataTable
+      dense={true}
+      columns={columns}
+      data={content}
+      noDataComponent={tableT('noRecords')}
+      highlightOnHover
+      pagination={content.length > 10}
+      paginationComponentOptions={{
+        rowsPerPageText: tableT('rowsPerPageText'),
+        rangeSeparatorText: tableT('rangeSeparatorText'),
+        selectAllRowsItem: true,
+        selectAllRowsItemText: tableT('selectAllRowsItemText'),
+      }}
+    />
   );
 }
 
 function transformContentToHTMLElement(contentRow: OverviewRow) {
   const badgeColor = getColorForResourceType(contentRow.resourceType);
   const titleColor = contentRow.isCategory ? 'nav-link' : '';
   return (
-    <Link href={contentRow.href} className="d-flex align-items-center text-dark text-decoration-none w-100 p-2">
+    <Link href={contentRow.href} className="d-flex align-items-center text-dark text-decoration-none w-100 px-2 py-4">
```

Comment:
Cant say I agree with these styling choices, I preferred the way it was before

Reply:
Matter of preference. I am fine to go with your version